tools/maint/rebaseline_tests.py

```diff
@@ -64,8 +64,13 @@ def process_changed_file(filename):
 def main(argv):
   parser = argparse.ArgumentParser()
   parser.add_argument('-s', '--skip-tests', action='store_true', help="don't actually run the tests, just analyze the existing results")
+  parser.add_argument('-b', '--new-branch', action='store_true', help='create a new branch containing the updates')
+  parser.add_argument('-c', '--clear-cache', action='store_true', help='clear the cache before rebaselining (useful when working with llvm changes)')
```

Comment:
Should this be the default perhaps?

Reply:
I think most of the time this script will be used to update expectations in response for local emscripten changes (i.e. minor changes to JS libraries) not upstream llvm change.   At least for me I often run rebases without clearing the cache when I know llvm has not changed.

Reply:
Fair enough. lgtm either way.